backend/x86_binary_emitter.ml

```diff
@@ -634,10 +634,12 @@ let emit_movq b ~dst ~src =
     emit_mod_rm_reg b no_rex [ 0x0F; 0x7E ] rm (rd_of_regf reg)
   | _ -> assert false
 
-let emit_movsd b dst src =
+let emit_mov_float ~width b dst src =
   match (dst, src) with
   | Regf reg, ((Regf _ | Mem _ | Mem64_RIP _) as rm) ->
-      buf_int8 b 0xF2;
+      (match width with
```

Comment:
I would probably factor out the
code emitting the prefixes, but
I reckon it is not systematically
done in existing code.

Reply:
Yeah, I didn't do that because the choice of prefix depends on the instruction (though only either 0x66/nothing or 0xF3/0xF2 here)